modules/food-menu/dietary-tags.js

```diff
@@ -12,31 +13,58 @@ const styles = StyleSheet.create({
 		flexDirection: 'row',
 		justifyContent: 'flex-end',
 	},
-	icons: {
+	wrapper: {
+		marginVertical: 5,
+	},
+	iconsList: {
 		marginHorizontal: 3,
 		width: 15,
 		height: 15,
 	},
+	iconsDetail: {
+		marginRight: 5,
+		width: 15,
+		height: 15,
+	},
 })
 
 export function DietaryTags({
```

Comment:
This should be turned into two components, not reused, IMO – a list vs an inline row of icons is too different to be contained within the same component.